src/tracer.ts

```diff
@@ -0,0 +1,30 @@
+import tracer from "dd-trace"
+
+export function init() {
+  const ddServicePrefix = "positron"
+
+  tracer.init({
+    // Setting the service name to  `.undetected` so that we detect services that
```

Comment:
Nice - how did you come across this pattern?

Reply:
Just via local development. Once I was able to connect to Data Dog for local development, it was relatively painless to rapidly prototype on the configuration layer.